script.js

```diff
@@ -1,4 +1,210 @@
+var draws = 0;
+var playerWins = 0;
+var computerWins = 0;
+var timesPlayed = 0;
+var previousWinner = '';
+
+var mode = 'waiting for playerName';
+var playerName = '';
+
+// generate a random number between 1 and 3
+var randomNumber = function () {
+  return (Math.floor(Math.random() * 3)) + 1;
+};
+
+// game block
+var playGame = function (playerGuess) {
+  var computerChoice = '';
+  var computerNumber = randomNumber();
+  var message = '';
+
+  if (computerNumber == 1) {
+    computerChoice = 'scissors';
+  } else if (computerNumber == 2) {
+    computerChoice = 'paper';
+  } else {
+    computerChoice = 'stone';
+  }
```

Comment:
This code to generate the computer choice can be abstracted to a helper function because how we generate the computer choice isn't relevant to the core game logic